plugins/GSdx/GSdx.rc

```diff
@@ -213,31 +214,32 @@ BEGIN
     LTEXT           "CRC Hack Level:",IDC_CRC_LEVEL_TEXT,22,208,79,8
     COMBOBOX        IDC_CRC_LEVEL,105,206,127,63,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     CONTROL         "Enable HW Hacks",IDC_HACKS_ENABLED,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,11,223,71,10
-    PUSHBUTTON      "Configure Hacks...",IDC_HACKSBUTTON,105,221,127,14
+    PUSHBUTTON      "Configure Hacks",IDC_HACKSBUTTON,105,221,127,14
```

Comment:
This makes it inconsistent with the "Configure Shade Boost..." button. Either undo this change or change the shade boost button as well.

Also, might be a good idea to provide screenshots of the dialogs.